### PR TITLE
Add OpenVINO-ONNX CI check

### PR DESCRIPTION
**Pull request by rblaczkowski**
Not yet completed - first try to add new CI check

### CODE REVIEW

File: .ci/openvino-onnx/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    """
+}
+
+def runContainerCommand(String command, String dir="${DOCKER_HOME}") {
+    sh """
+        docker exec -w ${dir} ${CONTAINER_ID} ${command}
+    """
+}
+
+def buildOpenVino() {
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:145]: Not needed at this point, please remove.

File: .ci/openvino-onnx/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+}
+
+
+def dockerPull(String imageName) {
+    sh """
+        docker pull ${imageName}:${BASE_IMAGE_TAG}
+    """
+}
+
+def runTests() {
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:127]: If we want to execute this manually, please remove `tox` command from Dockerfile.

File: .ci/openvino-onnx/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                            """,
+                    returnStdout: true
+                    ).trim()
+}
+
+def buildDockerImage() {
+    sh """
+        docker build --tag=openvino-onnx-ci  --file=.ci/openvino-onnx/Dockerfile \
+        --build-arg http_proxy=http://proxy-chain.intel.com:911/ \
+        --build-arg https_proxy=http://proxy-chain.intel.com:912/ --no-cache .
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:116]: Do we need `--no-cache` here?

File: .ci/openvino-onnx/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+// treaty provisions. No part of the Material may be used, copied, reproduced,
+// modified, published, uploaded, posted, transmitted, distributed, or disclosed
+// in any way without Intel's prior express written permission.
+// No license under any patent, copyright, trade secret or other intellectual
+// property right is granted to or conferred upon you by disclosure or delivery of
+// the Materials, either expressly, by implication, inducement, estoppel or
+// otherwise. Any license under such intellectual property rights must be express
+// and approved by Intel in writing.
+
+OPENVINO_REPO_ADDRESS = "https://github.com/postrational/openvino"
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:18]: We shouldn't hardcode `postrational` here.

File: .ci/openvino-onnx/Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+            }
+        }
+        stage("Build docker image") {
+            steps{
+                dir("${WORKDIR}") {
+                    buildDockerImage()
+                }
+            }
+        }
+        // Not yet implemented
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:205]: Remove commented out code.

File: .ci/openvino-onnx/Jenkinsfile [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+}
+
+pipeline {
+    agent {
+        label "OpenVino"
+    }
+    environment {
+        PROJECT_NAME = "openvino"
+        WORKDIR = "${WORKSPACE}/${BUILD_NUMBER}"
+        String imageName = "${DOCKER_REGISTRY}/ubuntu_20_04_onnx.dockerfile"
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:145]: This line seems unused.

File: .ci/openvino-onnx/Jenkinsfile [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+def buildDockerImage() {
+    sh """
+        docker build --tag=openvino-onnx-ci  --file=.ci/openvino-onnx/Dockerfile \
+        --build-arg http_proxy=http://proxy-chain.intel.com:911/ \
+        --build-arg https_proxy=http://proxy-chain.intel.com:912/ .
+    """
+}
+
+
+def dockerPull(String imageName) {
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:120]: This function seems unused.

File: .ci/openvino-onnx/Jenkinsfile [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+    """
+}
+
+def dockerRemove(String container_ID) {
+    sh """
+        docker rm -f ${container_ID} || true
+    """
+}
+
+def runContainerCommand(String command, String dir="${DOCKER_HOME}") {
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:132]: This function is unused.

File: .ci/openvino-onnx/Jenkinsfile [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+        """
+            git submodule init && git submodule update \
+                --init \
+                --no-fetch \
+                --recursive 
+        """
+    }
+}
+
+def runDockerContainer(String imageName) {
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:99]: This function is unused.

File: .ci/openvino-onnx/Jenkinsfile [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+                    Commit Hash: ${gitPrInfo.commitSubject}
+                    Commit Subject: ${gitPrInfo.commitHash}
+                    Jenkins Build: ${RUN_DISPLAY_URL}
+            """,
+            to: "${notifyPeople}"
+        )
+    }
+}
+
+def gitClone(String label, String address, String branch) {
```
Comment (postrational) [inline .ci/openvino-onnx/Jenkinsfile:68]: This function is unused.

File: .ci/openvino-onnx/Jenkinsfile [hunk 11]
```diff
@@ -0,0 +1,121 @@
+// INTEL CONFIDENTIAL
```
Comment (azhogov) [inline .ci/openvino-onnx/Jenkinsfile:1]: please replace license by Apache 2 here and below

**Pull request merged**